lib/sc_client.rb

```diff
@@ -278,16 +247,10 @@ def emitack(event, object, ack)
   # @param [String] channel A channel name
   #
   #
-  #
+  # @return [<Array>] <list of channels subscribed>
```

Comment:
We do not return any array here, it is just used to store list of channels. Remove this line.

Reply:
done